build.gradle

```diff
@@ -40,16 +35,17 @@ tasks.withType(JavaCompile) {
 	options.encoding = "UTF-8"
 }
 
-// Loom will automatically attach sourcesJar to a RemapSourcesJar task and to the "build" task
-// if it is present.
-// If you remove this task, sources will not be generated.
-task sourcesJar(type: Jar, dependsOn: classes) {
-	classifier = "sources"
-	from sourceSets.main.allSource
+java {
+	// Loom will automatically attach sourcesJar to a RemapSourcesJar task and to the "build" task
+	// if it is present.
+	// If you remove this line, sources will not be generated.
+	withSourcesJar()
 }
 
 jar {
-	from "LICENSE"
+	from("LICENSE") {
+		rename { "${it}_${project.archivesBaseName}"}
```

Comment:
??? It's not like we need to include any other licenses. If we did, I'd still call it `LICENSE` and any other licenses would be called `LICENSE-it`.

Reply:
I just updated it from the fabric-example-mod which had [changed that in 1.17](https://github.com/FabricMC/fabric-example-mod/commit/90cb78d4127924ad97f23f7221c68d3d345f1358). Should I revert it?

Reply:
Yes.